features/functionality/auth/login.feature

```diff
@@ -69,16 +68,15 @@ Feature: Log In
     Given sign up page of web application
     When I fill form on sign up page with new data
     And I submit form on sign up page
-    Then I should not be logged in the system
-    And I should be redirected to home page
+    Then I should be redirected to home page
+    And I should not be logged in the system
     And I should see following text on home page:
       """
       A message with a confirmation link has been sent to your email address. Please open the link to activate your account.
       """
     When I open login page
     And I fill form on login page
     And I submit form on login page
-    Then I should not be logged in the system
```

Comment:
Why this validation was removed?

Reply:
Because login page doesn't have menu section it the top anymore

Reply:
Then we need to add step When I visit home page Thten I should not be logged in the system